tango_simlib/model.py

```diff
@@ -1,22 +1,29 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
 #########################################################################################
 # Copyright 2017 SKA South Africa (http://ska.ac.za/)                                   #
 #                                                                                       #
 # BSD license - see LICENSE.txt for details                                             #
 #########################################################################################
+from __future__ import absolute_import, division, print_function
+
 import importlib
 import logging
 import sys
 import time
 import weakref
+from builtins import map, object, range
 from functools import partial
 
+from future import standard_library
+standard_library.install_aliases()
+
```

Comment:
Please fix imports as per [wiki](https://github.com/ska-sa/katdeveloper/wiki/Investigate-a-way-to-improve-Py3-migration-feedback-loop-on-tox#futurize-stage-2)